torch/quantization/fx/quantization_patterns.py

```diff
@@ -861,18 +870,37 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 "call_function to be a function instead of a string"
                 quantized_op = get_quantized_operator(node.target)
                 args = load_arg(quantized=[0])(node.args)
-                kwargs = {**load_arg(quantized=False)(node.kwargs), "output_scale": scale_arg, "output_zero_point": zero_point_arg}
+                kwargs = {**load_arg(quantized=False)(node.kwargs), "output_scale": scale_arg,
+                          "output_zero_point": zero_point_arg}
                 if quantized_op in ARGS_TO_SKIP:
                     args_to_skip = ARGS_TO_SKIP[quantized_op]
                     for arg in args_to_skip:
                         if arg in kwargs:
                             kwargs.pop(arg)
                 return quantizer.quantized_graph.create_node(
                     "call_function", quantized_op, args, kwargs)
+        elif is_reference and dtypes in [(torch.quint8, torch.qint8, None)]:
+            args = load_arg(quantized=[0])(node.args)
```

Comment:
is this line needed? would it get overriden by the next line

Reply:
i think its the load_arg statement that is doing something, I'm pulling this from the convrelu reference code so if its redundant then its also redundant there.